diff --git a/node_modules/create-file-webpack/index.d.ts b/node_modules/create-file-webpack/index.d.ts
new file mode 100644
index 0000000..037e577
--- /dev/null
+++ b/node_modules/create-file-webpack/index.d.ts
@@ -0,0 +1,15 @@
+import { Plugin } from "webpack";
+
+declare class CreateFileWebpack extends Plugin {
+	public constructor(options: CreateFileWebpack.CreateFileWebpackOptions);
+}
+
+declare namespace CreateFileWebpack {
+	type CreateFileWebpackOptions = {
+		path: string;
+		fileName: string;
+		content: string;
+	};
+}
+
+export = CreateFileWebpack;
